src/legacy/core_plugins/kibana/public/dashboard/grid/_dashboard_grid.scss

```diff
@@ -95,8 +103,25 @@
    */
   &.react-grid-placeholder {
     border-radius: $euiBorderRadius;
-    background: $euiColorPrimary;
+    background: $euiColorWarning;
   }
 }
 
+// When in view-mode only, and on tiny mobile screens, just stack each of the grid-items
+
+@include euiBreakpoint('xs') {
```

Comment:
We should ask what the PMs want for tablet. Could be arguable that it makes sense to do this there too?

Reply:
I'll bump it up to `xs, s` but that only goes to `767px`. I'm not sure dashboards are as broken past that breakpoint. It's really hard to judge because you don't know what size the user has optimized their layout for. Block layout-ing dashboards all the way up to `991px` I think could be unexpected and un-wanted.

Reply:
Yep. I think that's fine. Mostly just pointing out that under 768px it's still likely unreadilble, so may as well perform the same stacking.